package.json

```diff
@@ -27,8 +27,8 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
-    "ember-cli": "^2.0.0",
-    "execa": "^0.6.3",
+    "ember-cli": "^3.5.1",
+    "execa": "^1.0.0",
```

Comment:
was the `execa` update intended?

Reply:
yes it was: https://github.com/sindresorhus/execa/releases/tag/v1.0.0

> This marks execa as stable. No actual changes since 0.11.0.